port/common/omrsysinfo.c

```diff
@@ -902,6 +902,7 @@ omrsysinfo_cgroup_are_subsystems_enabled(struct OMRPortLibrary *portLibrary, uin
  * omrsysinfo_cgroup_enable_limits() before calling this function.
  * When the fuction returns OMRPORT_ERROR_SYSINFO_CGROUP_UNSUPPORTED_PLATFORM,
  * value of *limits is unspecified.
+ * Note that 'limit' parameter should not be NULL.
```

Comment:
Nitpick: `should not` -> `must not`  

This is a bit of a nitpick but `must` is a stronger requirement than `should`.  Since the code will assert it is not null, `must` is a better choice.